src/DataFixtures/ContentFixtures.php

```diff
@@ -71,11 +76,18 @@ private function loadContent(ObjectManager $manager): void
                 $content->setPublishedAt($this->faker->dateTimeBetween('-1 year'));
                 $content->setDepublishedAt($this->faker->dateTimeBetween('-1 year'));
 
+                $preset = $this->getPreset($contentType['slug']);
+
                 $sortorder = 1;
                 foreach ($contentType['fields'] as $name => $fieldType) {
                     $field = Field::factory($fieldType, $name);
                     $field->setName($name);
-                    $field->setValue($this->getValuesforFieldType($name, $fieldType));
+
+                    if (isset($preset[$name])) {
+                        $field->setValue((array) $preset[$name]);
```

Comment:
casting to array should be done inside setValue()

Reply:
Think so? Then we'd need to remove the typehint from `setValue`:

```php
    public function setValue(array $value): self
    {
        $this->value = $value;

        return $this;
    }
```

Reply:
Yes, that makes sense. Only Field class can we aware of that value is stored as array.

Reply:
Ok, will change it, and cast it to array in the method.